docs/arithmetic_expression.rst

```diff
@@ -45,6 +45,39 @@ The associativity and precedence of the overloaded Python arithmetic operators a
  * :code:`Int(1) + Int(2) + Int(3)` is equivalent to :code:`Add(Add(Int(1), Int(2)), Int(3))`
  * :code:`Int(1) + Int(2) * Int(3)` is equivalent to :code:`Add(Int(1), Mul(Int(2), Int(3)))` 
 
+Byteslice Arithmetic
+~~~~~~~~~~~~~~~~~~~~
+
+Byteslice arithemetic is available for Teal V4 and above. 
+Byteslice arithmetic operators allow up to 512-bit arithmetic.
+In PyTeal, byteslice arithmetic expressions include 
+:code:`TealType.Bytes` values as arguments (with the exception of :code:`BytesZero`)
+and must be 64 bytes or less.
+The table below summarizes the byteslize arithmetic operations in PyTeal.
+
+======================================= ============= ====================================================================== ==================
+Operator                                Return Type   Example                                                                Example Result
+======================================= ============= ====================================================================== ==================
+:any:`BytesLt(a, b) <BytesLt>`          :code:`Int`   :code:`BytesLt(Bytes("base16", "0xFF"), Bytes("base16", "0xFE"))`      :code:`0`
+:any:`BytesGt(a, b) <BytesGt>`          :code:`Int`   :code:`BytesGt(Bytes("base16", "0xFF"), Bytes("base16", "0xFE"))`      :code:`1`
+:any:`BytesLe(a, b) <BytesLe>`          :code:`Int`   :code:`BytesLe(Bytes("base16", "0xFF"), Bytes("base16", "0xFE"))`      :code:`0`
+:any:`BytesGe(a, b) <BytesGe>`          :code:`Int`   :code:`BytesGe(Bytes("base16", "0xFF"), Bytes("base16", "0xFE"))`      :code:`1`
+:any:`BytesAdd(a, b) <BytesAdd>`        :code:`Bytes` :code:`BytesAdd(Bytes("base16", "0xFF"), Bytes("base16", "0xFE"))`     :code:`0x1FD`
+:any:`BytesMinus(a, b) <BytesMinus>`    :code:`Bytes` :code:`BytesMinus(Bytes("base16", "0xFF"), Bytes("base16", "0xFE"))`   :code:`0x1`
+:any:`BytesMul(a, b) <BytesMul>`        :code:`Bytes` :code:`BytesMul(Bytes("base16", "0xFF"), Bytes("base16", "0xFE"))`     :code:`0xFD02`
+:any:`BytesDiv(a, b) <BytesDiv>`        :code:`Bytes` :code:`BytesDiv(Bytes("base16", "0xFF"), Bytes("base16", "0x11"))`     :code:`0xF`
+:any:`BytesMod(a, b) <BytesMod>`        :code:`Bytes` :code:`BytesMod(Bytes("base16", "0xFF"), Bytes("base16", "0x12"))`     :code:`0x3`
+:any:`BytesEq(a, b) <BytesEq>`          :code:`Int`   :code:`BytesEq(Bytes("base16", "0xFF"), Bytes("base16", "0xFF"))`      :code:`1`
+:any:`BytesNeq(a, b) <BytesNeq>`        :code:`Int`   :code:`BytesNeq(Bytes("base16", "0xFF"), Bytes("base16", "0xFF"))`     :code:`0`
```

Comment:
The examples and results are nice! One formatting issue though: hexadecimal byte strings need to be even lengths (since 2 characters define 1 byte), so could you prefix the odd-length example results with 0, e.g. `0x3` -> `0x03`.

Also, could you move the rows for `BytesEq` and `BytesNeq` right below `BytesGe`? I think it makes sense if all the comparison ops are next to each other.

Reply:
Good catch!